src/features/stake/components/StakingConfirmationTx/Deposit.tsx

```diff
@@ -20,6 +20,8 @@ const CURRENCY = 'USD'
 const StakingConfirmationTxDeposit = ({ order }: StakingOrderConfirmationViewProps) => {
   const isOrder = order.type === ConfirmationViewTypes.KILN_NATIVE_STAKING_DEPOSIT
 
+  // the fee is returned in decimal format, so we multiply by 100 to get the percentage
+  const fee = (order.fee * 100).toFixed(2)
```

Comment:
shall we put it into a utility function, in order to re-use that in future scenarios?

Reply:
I thought about this, but don't want to "optimize prematurely". That's the only place where we are showing this right now (as far as I remember). If we need it later, we can extract it.